include/deal.II/matrix_free/evaluation_kernels.h

```diff
@@ -3703,41 +3714,47 @@ namespace internal
                                           (fe_degree == -1) ? 1 : fe_degree + 1,
                                           0,
                                           Number,
-                                          0>,
+                                          0,
+                                          Number2>,
         EvaluatorTensorProductAnisotropic<evaluate_raviart_thomas,
                                           dim,
                                           (fe_degree == -1) ? 1 : fe_degree,
                                           0,
                                           Number,
-                                          0>>::type;
+                                          0,
+                                          Number2>>::type;
       using Evalf1 = typename std::conditional<
         face_direction == 1,
         EvaluatorTensorProductAnisotropic<evaluate_raviart_thomas,
                                           dim,
                                           (fe_degree == -1) ? 1 : fe_degree + 1,
                                           0,
                                           Number,
-                                          1>,
+                                          1,
+                                          Number2>,
         EvaluatorTensorProductAnisotropic<evaluate_raviart_thomas,
                                           dim,
                                           (fe_degree == -1) ? 1 : fe_degree,
                                           0,
                                           Number,
-                                          1>>::type;
+                                          1,
+                                          Number2>>::type;
       using Evalf2 = typename std::conditional<
         face_direction == 2,
         EvaluatorTensorProductAnisotropic<evaluate_raviart_thomas,
                                           dim,
                                           (fe_degree == -1) ? 1 : fe_degree + 1,
                                           0,
                                           Number,
-                                          2>,
+                                          2,
+                                          Number2>,
```

Comment:
Wouldn't it make sense to sort the parameters? This class is rather new and probably no one uses this internal class.

Reply:
Yes, sorting the arguments makes a lot of sense (albeit I did not add any type here and rather just changed from a default template argument).

Reply:
Yes, sorting the arguments makes a lot of sense (albeit I did not add any type here and rather just changed from a default template argument).

Reply:
I see. I guess also approved that last year^^